### ISSUE TITLE
Is the accuracy on the test set in the figure 41.8% or 100%?

### ISSUE THREAD

**Issue by szpxmu** [SHI-Labs/Compact-Transformers#81]
![7ee1e9dc0b16bb896c42883a297a950](https://github.com/SHI-Labs/Compact-Transformers/assets/12219867/4bcd324d-9e78-4702-8645-52757bfaeca3)

**Comment by stevenwalton** [comment]
Definitely not 100%. You're at 42%. This is just output from the timm training script. See [their repo](https://github.com/huggingface/pytorch-image-models) for more details.

**Comment by dreamillusion** [comment]
![lipu](https://github.com/SHI-Labs/Compact-Transformers/assets/77829685/d187fa72-b41f-4d93-8a64-f7f3bb29592e)
Is my model's best metric accuracy, and why is it so high?
